WalletWasabi.Fluent/Models/Wallets/Address.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.Clustering;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Extensions;
+using WalletWasabi.Hwi;
+using WalletWasabi.Logging;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Models.Wallets;
+
+public class Address : ReactiveObject, IAddress
```

Comment:
We might rename this to AddressModel.

Reply:
@SuperJMN I'd rather not.

The only reason I've used `WalletModel` is because `Wallet` and `IWallet` already exist.